From 0a33fff0fc248166babdf120dc17b60a1f12b2d3 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Mon, 8 Apr 2024 18:42:55 +0200
Subject: [PATCH 0813/1695] dts: stm32: set tampers as a wakeup source for
 stm32mp257f-ev1

Configure EXTI2 as TAMP wakeup parent.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 14547111f5054e43543043c8133208a0f1955fc2 dts: stm32: set tampers as a wakeup source for stm32mp257f-ev1]
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: Ic3f6a7004b1a0a48ae3ced1baec6c3b9c106072e
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/372211
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 core/arch/arm/dts/stm32mp257f-ev1.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp257f-ev1.dts b/core/arch/arm/dts/stm32mp257f-ev1.dts
index e2c8c5eed..dd23200d0 100644
--- a/core/arch/arm/dts/stm32mp257f-ev1.dts
+++ b/core/arch/arm/dts/stm32mp257f-ev1.dts
@@ -487,6 +487,8 @@
 
 &tamp {
 	status = "okay";
+	wakeup-source;
+	wakeup-parent = <&exti2>;
 	st,tamp-passive-precharge = <2>;
 	st,tamp-passive-nb-sample = <4>;
 	st,tamp-passive-sample-clk-div = <16384>;
-- 
2.47.2

